outputs.tf

```diff
@@ -13,37 +13,31 @@ output "hostname" {
   value       = "${var.deploy_id}.${var.route53_hosted_zone_name}"
 }
 
-output "efs_volume_handle" {
-  description = "EFS volume handle <filesystem id id>::<accesspoint id>"
-  value       = module.storage.efs_volume_handle
+output "efs_access_point" {
+  description = "EFS access point"
+  value       = module.storage.efs_access_point
 }
 
-output "efs_access_point_id" {
-  description = "EFS access_point id"
-  value       = module.storage.efs_access_point_id
-}
-
-output "efs_file_system_id" {
-  description = "EFS filesystem id"
-  value       = module.storage.efs_file_system_id
-}
-
-output "region" {
-  description = "Deployment region."
-  value       = var.region
-}
-
-output "deploy_id" {
-  description = "Deployment ID."
-  value       = var.deploy_id
+output "efs_file_system" {
+  description = "EFS file system"
+  value       = module.storage.efs_file_system
 }
 
 output "s3_buckets" {
-  description = "S3 buckets name,arn."
+  description = "S3 buckets"
   value       = module.storage.s3_buckets
 }
 
+output "domino_key_pair" {
```

Comment:
Do we need this?

Reply:
got rid of yours since we only needed one.